assignments/objects.js

```diff
@@ -11,54 +11,176 @@
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
 
-// Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+const kennan = {
+	id: 2,
+	name: 'Kennan',
+	email: 'kdiben1@tinypic.com',
+	gender: 'M',
+	speak: function() {
+		console.log('Hello, my name is Kennan!');
+	},
+};
+console.log(kennan.speak());
 
-// === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
+// Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 
-// ==== Stretch Challenge: Nested Objects and the this keyword ==== 
+const antonietta = {
+	id: 5,
+	name: 'Antonietta',
+	email: 'adaine5@samsung.com',
+	gender: 'F',
+	multiplyNums: function(a, b) {
+		return a * b;
+	},
+};
+console.log(antonietta.multiplyNums(3, 4));
 
-// 1. Create a parent object with properties for name and age.  Make the name Susan and the age 70.
-// 2. Nest a child object in the parent object with name and age as well.  The name will be George and the age will be 50.
-// 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
-// 4. Give each of the objects the ability to speak their names using the this keyword.
+// === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
 
-const parent = {}
+// ==== Stretch Challenge: Nested Objects and the this keyword ====
+
+// 1. Create a parent object with properties for name and age. Make the name Susan and the age 70.
+const parent = {
+	name: 'Susan',
+	age: 70,
+};
+
+// 2. Nest a child object in the parent object with name and age as well. The name will be George and the age will be 50.
+const parent = {
+	name: 'Susan',
+	age: 70,
+	child: {
+		name: 'George',
+		age: 50,
+	},
+};
+// 3. Nest a grandchild object in the child object with properties for name and age. The name will be Sam and the age will be 30
+const parent = {
+	name: 'Susan',
+	age: 70,
+	child: {
+		name: 'George',
+		age: 50,
+		grandchild: {
+			name: 'Sam',
+			age: 30,
+		},
+	},
+};
+// 4. Give each of the objects the ability to speak their names using the this keyword
+const parent = {
+	name: 'Susan',
+	age: 70,
+	speak: function() {
+		return 'My name is ' + this.name;
+	},
+	child: {
+		name: 'George',
+		age: 50,
+		speak: function() {
+			return 'My name is ' + this.name;
+		},
+		grandchild: {
+			name: 'Sam',
+			age: 30,
+			speak: function() {
+				return 'My name is ' + this.name;
+			},
+		},
+	},
+};
 
 // Log the parent object's name
+console.log(parent.name);
 
 // Log the child's age
+console.log(parent.child.age);
 
 // Log the name and age of the grandchild
+console.log(parent.child.grandchild);
```

Comment:
Remember to single out the name and age of the grandchild here. Right now, you are logging out the entire object.